deepcell/model_zoo/tracking.py

```diff
@@ -287,7 +310,7 @@ def __init__(self,
         # Create model
         self.training_model, self.inference_model = self.get_models()
 
-    def get_embedding_temporal_merge_model(self, merge_type='cnn'):
+    def get_embedding_temporal_merge_model(self, merge_type='lstm'):
```

Comment:
If only LSTM works, let's just remove the "cnn" option and `merge_type` parameter entirely.

Reply:
Same with `get_delta_temporal_merge_model`.